apps/custom-ui-floating-window-demo/pages/api/getContact.js

```diff
@@ -0,0 +1,77 @@
+import { PersonsApi } from 'pipedrive';
+import logger from '../../shared/logger';
+import { getAPIClient } from '../../shared/oauth';
+const log = logger('Get Contact API 📕');
+/**
+ * Get the current session
+ * Obtain the person by ID / Number
+ * Get associated deals of that person (if available)
+ * Return the response
+ */
+const handler = async (req, res) => {
```

Comment:
Can you rename these files under /api to follow REST practises?
These file names will be equal to the following API routes
Old: `getContact.js`, new:  `contacts/[id].js` -> `GET /api/contacts/{id}`
Old: `getContacts.js` new: `contacts.js` -> `GET /api/contacts`
Old: `addContact.js` new: `contacts.js` -> `POST /api/contacts` 
Old: `addNotesToContact.js` new: `contacts/[id].js` -> `PUT /api/contacts/{id}`
Old: `addCallLogsToDeal.js` new: `deal/[id].js` -> `PUT /api/deal/{id}`

For this you will need to merge some files so best way for it is to check request method
pseudo:
```
if request method == get
    handle get request
if request method === post
    handle post request
```

Reply:
That is a really good suggestion. Since this is a demo app, is it okay to keep it as such for simplicity? Perhaps I can mention this in the codelabs / guide